resources/msg.go

```diff
@@ -69,6 +69,16 @@ func (obj *MsgRes) Validate() error {
 			return fmt.Errorf("fields cannot begin with _")
 		}
 	}
+        ValidPriorities := []string{"Emerg", "Alert", "Crit", "Err", "Warning", "Notice", "Info", "Debug"}
+        priorityIsValid := false
+        for _, element := range ValidPriorities {
+            if strings.Compare(obj.Priority, element) == 0 {
```

Comment:
Please read the recommendation on this methods docs: https://golang.org/pkg/strings/#Compare